### ISSUE TITLE
ReferenceError: Unknown plugin "rewire"

### ISSUE THREAD

**Issue by burabure** [KeywordBrain/redbox-react#34]
Updated to 1.1.1 from 1.1.0 and now i'm getting this error in my webpack build

```
ERROR in ./~/redbox-react/lib/index.js
Module build failed: ReferenceError: Unknown plugin "rewire"
    at PluginManager.subnormaliseString (/home/burabure/proyectos/slm-cotizaciones/node_modules/babel-core/lib/transformation/file/plugin-manager.js:147:13)
    at PluginManager.add (/home/burabure/proyectos/slm-cotizaciones/node_modules/babel-core/lib/transformation/file/plugin-manager.js:190:40)
    at File.buildTransformers (/home/burabure/proyectos/slm-cotizaciones/node_modules/babel-core/lib/transformation/file/index.js:237:21)
    at new File (/home/burabure/proyectos/slm-cotizaciones/node_modules/babel-core/lib/transformation/file/index.js:139:10)
    at Pipeline.transform (/home/burabure/proyectos/slm-cotizaciones/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at transpile (/home/burabure/proyectos/slm-cotizaciones/node_modules/babel-loader/index.js:12:22)
    at Object.module.exports (/home/burabure/proyectos/slm-cotizaciones/node_modules/babel-loader/index.js:71:12)
 @ ./app/containers/App.jsx 13:19-42
```

**Comment by ntdb** [comment]
+1

**Comment by gaearon** [comment]
This should be fixed by #45. See https://github.com/KeywordBrain/redbox-react/pull/37#issuecomment-178322170 for an explanation why this happened.
